bundles/org.openhab.binding.unifi/README.md

```diff
@@ -287,3 +354,23 @@ sitemap unifi label="UniFi Binding"
 	}
 }
 ```
+
+### `rule actions`
+
+```java
+val uniFiActions = getActions("unifi","unifi:site:mycontroller:mysite")
```

Comment:
Sorry, late finding: Since these actions are tied to the `site` thing, can this thing also be added to the example under chapter `unifi.things`? Then it would probably be "controller" rather than "mycontroller" for consistency with example.

Reply:
I made the controller id consistent with the example (home), but I doubt this makes the whole example any clearer.
I also don't think this should be moved into the things example section, as that purely shows syntax for a things file. And so do the examples in the others sections for their respective files. Rule actions don't go in things files, but in rules files for dsl (or are used in any of the other scripting languages). So I think it should be kept at the end.

Reply:
Intention of my comment was simply to add "mysite" to the things example, so the example would be complete, i.e. "mysite" would refer to something also shown in the full example.

So the things example would be something like:

```
Bridge unifi:controller:home "UniFi Controller" [ host="unifi", port=8443, unifios=false, username="$username", password="$password", refresh=10 ] {
	Thing wirelessClient matthewsPhone "Matthew's iPhone" [ cid="$cid", site="default", considerHome=180 ]
	Thing site mysite "My Site" [ sid="$sid" ]
}
```

Reply:
OK, clearer. I added it in.